### DISCUSSION TITLE
8080 Freezes after starting Run

### DISCUSSION THREAD

**Discussion by Gmanxxx** [thehighnibble/thehighnibble.github.io#125] [IMSAI8080esp replica]
I am building the 8080 and at the point of powering up the 8080 as shown in the video #19 and have two issues:

1. When I power up the 8080, there is a delay of 2-3 seconds before the LED light.  This may not be a problem as most of that time is the ESP sending out text to the PC terminal emulator.

2. When I put the 8080 in Run it freezes.  I can't put it back in Halt, and if I press the "Off" switch, nothing happens.  I have to reset the ESP to unlock it. 

In the boot sequence, there is one red error message "E (1046) nvs: Error (ESP_ERR_NVS_NOT_FOUND) opening NVS handle!"  I have tripled checked the ICs and solder joints and everything seems OK.

**Comment by Gmanxxx** [comment]
Here is the log from the ESP32 for Boot, 8080 Power on, and 8080 Power off in case there is something useful there.
[8080.log](https://github.com/thehighnibble/thehighnibble.github.io/files/13508350/8080.log)

**Comment by Gmanxxx** [comment]
Issue #2 is resolved after I initialized and ran CP/M.  I am assuming there was some issue with the random bytes before being initialized.  Item #1 is still there, is a 2-3 second delay after turning on the soft power button normal?

**Reply by nbreeden2** [reply]
It sounds like the NVRAM (used by the emulation) wasn't initialized causing the lock up. When you used the front panel to configure for CP/M the NVRAM was initialized.
I believe the delay after power on is normal and is most likely a quirk of the emulation, Dave (High Nibble) may correct me on this.

**Comment by Gmanxxx** [comment]
Assuming the delay is due to the simulation startup.
